packaging/redhat/cvmfs-config-osg.spec

```diff
@@ -0,0 +1,120 @@
+Summary: CernVM File System OSG Configuration and Public Keys
+Name: cvmfs-config-osg
+Version: 2.0
+Release: 2%{?dist}
+# download with:
+# $ curl -L -o cvmfs-config-osg-%{version}.tar.gz \
+#   https://github.com/opensciencegrid/cvmfs-config-osg/archive/v%{version}.tar.gz
+Source: %{name}-%{version}.tar.gz
+BuildArch: noarch
+Group: Applications/System
+License: BSD
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+Provides: cvmfs-config = %{version}-%{release}
+Obsoletes: cvmfs-keys < 1.6
+Provides: cvmfs-keys = 1.7
+Obsoletes: cvmfs-init-scripts < 1.0.21
+Provides: cvmfs-init-scripts = 1.0.22
+
+Obsoletes: oasis-config < 8
+Provides: oasis-config = 9
+
+Conflicts: cvmfs-config-default
+
+Conflicts: cvmfs < 2.3.3
+Conflicts: cvmfs-server < 2.3.3
+
+%prep
+%setup
+
+%description
+Default configuration parameters and public keys for CernVM-FS
+
+%install
+rm -rf $RPM_BUILD_ROOT
+for cvmfsdir in keys/opensciencegrid.org config.d default.d; do
+    mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/cvmfs/$cvmfsdir
+done
+for key in opensciencegrid.org.pub; do
+    install -D -m 444 "${key}" $RPM_BUILD_ROOT%{_sysconfdir}/cvmfs/keys/opensciencegrid.org
```

Comment:
Should the install get wrapped up into a `Makefile`?  I.e., have two targets - one RedHat, one Debian?